src/analysis/retail/deathknight/shared/talents/SoulReaper.tsx

```diff
@@ -77,6 +88,137 @@ class SoulReaper extends ExecuteHelper {
+        <div style={{ display: 'flex', alignItems: 'center', padding: '10px' }}>
+          <div style={{ marginRight: '20px' }}>
+            <SpellIcon spell={TALENTS.SOUL_REAPER_TALENT} />
+          </div>
+          <div style={{ flex: 1 }}>
+            <div style={{ marginBottom: '8px', fontSize: '16px', fontWeight: 'bold' }}>
+              Soul Reaper Efficiency
+            </div>
+            <div
+              style={{
+                position: 'relative',
+                height: '24px',
+                backgroundColor: '#1a1a1a',
+                borderRadius: '12px',
+                overflow: 'hidden',
+              }}
+            >
+              <div style={{ position: 'absolute', width: '100%', height: '100%', display: 'flex' }}>
+                <div
+                  style={{
+                    width: '70%',
+                    backgroundColor: '#2a2a2a',
+                    borderRight: '1px solid #444',
+                  }}
+                />
+                <div
+                  style={{ width: '15%', backgroundColor: '#333', borderRight: '1px solid #444' }}
+                />
+                <div
+                  style={{
+                    width: '10%',
+                    backgroundColor: '#3a3a3a',
+                    borderRight: '1px solid #444',
+                  }}
+                />
+                <div style={{ width: '5%', backgroundColor: '#444' }} />
+              </div>
+              <div
+                style={{
+                  position: 'absolute',
+                  width: `${efficiency}%`,
+                  height: '100%',
+                  background: `linear-gradient(90deg, ${barColor} 0%, ${barColor}dd 100%)`,
+                  boxShadow: `0 0 10px ${barColor}66`,
+                  transition: 'width 0.3s ease',
+                }}
+              />
+              {[70, 85, 95].map((pct) => (
+                <div
+                  key={pct}
+                  style={{
+                    position: 'absolute',
+                    left: `${pct}%`,
+                    top: 0,
+                    width: '2px',
+                    height: '100%',
+                    backgroundColor: 'rgba(255,255,255,0.3)',
+                  }}
+                />
+              ))}
+            </div>
+            <div
+              style={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                alignItems: 'center',
+                marginTop: '8px',
+                fontSize: '14px',
+              }}
+            >
+              <div>
+                <TooltipElement
+                  content={`${actual} Soul Reaper casts in execute phase out of ${possible} total`}
+                >
+                  <span style={{ color: '#aaa' }}>
+                    {actual} / {possible} procced ({efficiency.toFixed(1)}%)
+                  </span>
+                </TooltipElement>
+              </div>
+              <div>
```

Comment:
this results in the performance mark being placed too far away on larger screens (screenshot on my work pc below), causing it to be visually disconnected from the stat. should just move the mark to precede the text as is conventional elsewhere

![image](https://github.com/user-attachments/assets/1e14483e-aa90-4197-af36-4b65fce190c8)